### PR TITLE
DX-2849 Add `webrtc_utils`

### PR DESCRIPTION
**Pull request by ajrice6713**
Add WebRTC Utilities module with helper methods to generate transfer BXML

### CODE REVIEW

File: bandwidth/utilities/web_rtc.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    Args:
+        device_token (str): The device token.
+        voice_call_id (str): The Bandwidth voice Call ID.
+        sip_uri (str, optional): The SIP URI to transfer the call to. Defaults to 'sip:sipx.webrtc.bandwidth.com:5060'.
+
+    Returns:
+        str: <Transfer> BXML Verb
+    """
+    uui = "".join(voice_call_id.split("-")[1::])
```
Comment (carrington-bw) [inline bandwidth/utilities/web_rtc.py:16]: You could make a private helper to generate the Transfer and reuse it in this function and in generate_transfer_bxml

File: bandwidth/utilities/web_rtc.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+from bandwidth.model.bxml.response import Response
+from bandwidth.model.bxml.verbs import *
+
+
+def _generate_transfer_verb(device_token: str, voice_call_id: str, sip_uri: str = 'sip:sipx.webrtc.bandwidth.com:5060') -> Transfer:
```
Comment (carrington-bw) [inline bandwidth/utilities/web_rtc.py:5]: For better or worse, iIn the SDK for all of our other languages the generate_transfer_verb() equivalent returns the BXML. This is the node version: https://github.com/Bandwidth/node-webrtc/blob/main/src/controllers/apiController.ts#L345


To make it consistent across languages perhaps name this one something like generate_transfer_verb_model, generate_transfer_bxml would become generate_transfer_bxml_verb, and generate_transfer_bxml_document would become generate_transfer_bxml

Comment (ajrice6713) [reply bandwidth/utilities/web_rtc.py:5]: This version of the SDK Will be breaking - so we could take the opprotunity to standardize on new names for all of the languages. We will be writing migration documentation for each language before going live. 

Im fine with 
```py
_generate_transfer_model()
generate_transfer_bxml_verb()
generate_transfer_bxml()
```
if you guys are happy

Comment (carrington-bw) [reply bandwidth/utilities/web_rtc.py:5]: Sounds good to me

**Review by brianluisgomez — APPROVED** [review]

**Review by ckoegel — APPROVED** [review]

**Pull request merged**